README.md

```diff
@@ -1 +1,27 @@
-Poort8.Ishare.Service
\ No newline at end of file
+Poort8.Ishare.Service
+
+
+# Testing
+
+The Poort8.Ishare.Service container can be tested using a the Postman test collection `Poort8.Ishare.Service.postman_collection.json`. After obtaining an iSHARE test certificate, one can directly try the Poort8 implementation of Poort8.Ishare.Common and Poort8.Ishare.Service.
+
+Then, by changing the collection variables, one can use this Postman collection to test your own implementation.
+
+
+### How does one use it?
+
+1. [Get Postman](https://www.getpostman.com/apps)
+2. Run it. Don't bother signing in if you don't want to, there's a small link on the bottom to skip. This project does not use any of Postman's cloud features.
+3. Click `Import` button in top left and drag `Poort8.Ishare.Service.postman_collection.json there.
+4. Open the collection `Sample Service Provider Calls` and go to the tab `Variables`. Replace serviceConsumer.EORI with the EORI number from the iSHARE test certificate in the format `EU.EORI.NL_________`.
+5. Also in the tab `Variables`, add your iSHARE public and private key in the designated variables. This is used in the test collection for obtaining the iSHARE required client assertion. NB. this is not good practice for one's operational implementation. Therefore ONLY do this with test certificates, do not add the private key from any operational certificate. 
```

Comment:
Explain that the private key of your ishare test certificate is sent to the ishare scheme owner endpoint in order to sign a client assertion? Or maybe this is too much detail..?